packages/webapp/src/locales/i18n.js

```diff
@@ -12,7 +12,7 @@ i18n
     defaultNS: 'translation',
     nsSeparator: ':',
     fallbackLng: 'en',
-    locales: ['en', 'pt', 'es', 'fr'],
+    locales: ['en', 'pt', 'es', 'fr', 'de', 'hi', 'pa', 'ml'],
```

Comment:
Could we get the array of languages here and in the `i18next-parser` file from the `supportedLanguages` array? I love how you minimized having to have a list of the languages in a thousand places and thinking if we could extend that further.